File: LMS_CMS_Frontend/src/app/Models/LMS/assignment-question.ts
import { QuestionBank } from "./question-bank";


export class AssignmentQuestion {
      constructor(
        public id: number = 0,
        public assignmentID: number = 0,
        public questionBankID: number = 0,
        public questionBank: QuestionBank ,
        public insertedByUserId: number = 0,
    ) {}
}

File: LMS_CMS_Backend/LMS_CMS_DAL/Models/Domains/LMS/AssignmentQuestion.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LMS_CMS_DAL.Models.Domains.LMS
{
    public class AssignmentQuestion : AuditableEntity
    {
        [Key]
        public long ID { get; set; }

        [ForeignKey("Assignment")]
        public long AssignmentID { get; set; }
        public Assignment Assignment { get; set; }

        [ForeignKey("QuestionBank")]
        public long QuestionBankID { get; set; }
        public QuestionBank QuestionBank { get; set; }
    }
}
